tests/tracking/test_rest_tracking.py

```diff
@@ -1047,16 +1047,58 @@ def get(self, key, default=None):
 def test_create_model_version_with_local_source(mlflow_client):
```

Comment:
@harupy Can we also add a test for the case where a user starts the MLflow server with `--default-artifact-root <local_path>` and then a client does `mlflow.sklearn.log_model(..., registered_model_name="mymodel")`? That way, we can catch this kind of regression in the future. Right now, we're missing coverage.

Reply:
Added!